### ISSUE TITLE
When CSS styling turned off, menu is not accessible. (colour contrast)

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#113]
This is to track the conversation we had in testing channel in Slack.
![image](https://cloud.githubusercontent.com/assets/15111528/10996191/fb7601d6-84d6-11e5-9958-3a177e09859d.png)

**Comment by christosptr** [comment]
@HerinHentry can you please let me know how you do that so i can reproduce

**Comment by HerinHentry** [comment]
@christosptr I used the Vision Australia tool bar to disable styles. This issue has been resolved as the menu went thru some changes. Closing issue.
